data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/regenerate/regenerate_no_poison.cfg

```diff
@@ -0,0 +1,74 @@
+#####
+# API(s) being tested: [regenerate]poison=
+##
+# Actions:
+# Give the leaders a regenerate ability with no poison attribute.
+# Wait a turn for regenerating to take place.
+##
+# Expected end state:
+# The Mage is poisoned but still has full hp.
+#####
+{GENERIC_UNIT_TEST "regenerate_no_poison" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    [regenerate]
+                        id = "test-regenerate"
+                        name = _"test-regenerate"
```

Comment:
This one is probably leaking into the `wesnoth` textdomain.

I've just run a pot-update on master, and found these two from the previous PR are leaking into `wesnoth`:
```
#: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heal_no_poison.cfg:25
#: data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/heals/heal_no_value.cfg:25
msgid "test-heals"
```

Reply:
Updated.